odl/util/testutils.py

```diff
@@ -298,11 +298,12 @@ class FailCounter(object):
     ``*** FAILED 1 TEST CASE(S) ***``
     """
 
-    def __init__(self, test_name, err_msg=None):
+    def __init__(self, test_name, err_msg=None, quiet=False):
```

Comment:
This should clearly be called `verbose` as well, so that we dont need to do the `quiet = not verbose` all over the board.